.kitchen.yml

```diff
@@ -15,14 +15,33 @@ provisioner:
         - docker
 
 platforms:
-  - name: ubuntu-14.04
+  - name: ubuntu-16.04
+    driver_config:
+      provision_command:
+        - apt-get update
+      run_command: /lib/systemd/systemd
   - name: debian-jessie
     driver_config:
+      provision_command:
+        - apt-get update
       run_command: /sbin/init
```

Comment:
Mind you change this `run_command` to `/lib/systemd/systemd`, for consistency?

Reply:
Sure, good catch @javierbertoli